pkg/build/linux_test.go

```diff
@@ -48,19 +50,26 @@ func enumerateFlags(t *testing.T, flags, allFlags []string) {
 	}
 	t.Run(strings.Join(flags, "-"), func(t *testing.T) {
 		t.Parallel()
-		sign1, sign2 := "", ""
-		var wg sync.WaitGroup
-		wg.Add(2)
-		go func() {
-			sign1 = sign(t, flags, false, false)
-			wg.Done()
-		}()
-		go func() {
-			sign2 = sign(t, flags, false, true)
-			wg.Done()
-		}()
-		sign3 := sign(t, flags, true, false)
-		wg.Wait()
+		sign1, sign2, sign3 := "", "", ""
+		g, _ := errgroup.WithContext(context.Background())
+		g.Go(func() error {
+			var err error
+			sign1, err = sign(t, flags, false, false)
+			return err
+		})
+		g.Go(func() error {
+			var err error
+			sign2, err = sign(t, flags, false, true)
+			return err
+		})
+		g.Go(func() error {
+			var err error
+			sign3, err = sign(t, flags, true, false)
+			return err
+		})
+		if err := g.Wait(); err != nil {
+			t.Error(err)
+		}
 		if sign1 != sign2 {
 			t.Errorf("signature has changed after a comment-only change")
```

Comment:
Formatting isn't needed here. t.Error() instread of t.Errorf() will also work.
But it is not a deal changer. Feel free to ignore.

Reply:
I think it'd still make debugging easier, so I think we'd better keep it. But yes, for tests it's not that critical.